.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Something isn't working as expected
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- NOTE:
+
+If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
+
+If you want help on your bug, please also send us the github repository where your hexo code is stored. If it is not on github, please post it on github, it would greatly help.
```

Comment:
Must be github? I personally don't mind if reporter is using others (gitlab, bitbucket, etc), as long it's git and public.

Reply:
I just copied this sentence from the current issue template. I will fix. Thanks :)